pkg/build/api/v1/types.go

```diff
@@ -780,6 +811,13 @@ type BuildTriggerPolicy struct {
 
 	// imageChange contains parameters for an ImageChange type of trigger
 	ImageChange *ImageChangeTrigger `json:"imageChange,omitempty" protobuf:"bytes,4,opt,name=imageChange"`
+
+	// GitLabWebHook contains the parameters for a GitLab webhook type of trigger
+	GitLabWebHook *WebHookTrigger `json:"gitlab,omitempty" protobuf:"bytes,5,opt,name=gitLab"`
```

Comment:
for github we did:
github

so this should be gitlab for both the json and proto name.